nautobot/utilities/templates/utilities/templatetags/utilization_graph.html

```diff
@@ -1,7 +1,9 @@
-<div class="progress text-center">
+<div {% if utilization_count and total_count %}title="Used: {{ utilization_count }}&#013Total Count: {{ total_count }}"
```

Comment:
`&#013` should perhaps be `&#13;` (note semicolon). Does this display as intended in Firefox, Chrome, and Safari (IIRC you're an MS Edge user, so add that to the list as well :-) )?

Reply:
I made the change, but what was there did work in Safari, Firefox, and chromium based browsers. It mimicked another combination elsewhere in the code that I'm not remembering specifically now.